colibre/auto_plotter/cold_gas_relations.yml

```diff
@@ -22,41 +22,18 @@ stellar_mass_molecular_to_neutral_fraction_30:
     end:
       value: 1e12
       units: Solar_Mass
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
```

Comment:
I'm still not very happy with this... I would prefer it if we're not including galaxies in the median line then there should be a 'physical' reasoning for it, e.g. we only want to include SFing galaxies